From f36dfb7ec1c780bdb74a4879fcce4be63adbaa6e Mon Sep 17 00:00:00 2001
From: Jan Janssen <medhefgo@web.de>
Date: Fri, 27 Jan 2023 14:28:58 +0100
Subject: [PATCH] meson: Properly install 90-uki-copy.install

(cherry picked from commit 4c181c1a33ef4de0130a131a2b332348dda672ed)

Resolves: RHEL-16354
---
 src/kernel-install/meson.build | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/kernel-install/meson.build b/src/kernel-install/meson.build
index 68a4d43862..2ff62d5935 100644
--- a/src/kernel-install/meson.build
+++ b/src/kernel-install/meson.build
@@ -3,10 +3,13 @@
 kernel_install_in = files('kernel-install.in')
 loaderentry_install = files('90-loaderentry.install')
 
-uki_copy_install = files('90-uki-copy.install')
+kernel_install_files = files(
+        '50-depmod.install',
+        '90-uki-copy.install',
+)
 
 if want_kernel_install
-        install_data('50-depmod.install',
+        install_data(kernel_install_files,
                      loaderentry_install,
                      install_mode : 'rwxr-xr-x',
                      install_dir : kernelinstalldir)
